docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,83 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+If you have [configured the frame system](/services/frame-system/#configuration) for the components of your robot, then you can implement this model to identify well separated objects above a flat plane.
+
+To configure an `obstacles_depth` segmenter, first decide if you want to use the segmenter with or without intrinsic parameters.
+
+This is determined by the value of the configuration attribute `with_geometries`:
```

Comment:
```suggestion
To configure an `obstacles_depth` segmenter, first decide if you want to use the segmenter with or without intrinsic parameters and configure the `with_geometries` attribute accordingly:
```
I think it's unclear what "This" is and I don't like the passive voice. Is this still correct?

Reply:
Still correct, thank you for the suggestion and cleaning up the passive voice! Done